### ISSUE TITLE
GraphQLDefinitionsFactory generate empty output on windows 

### ISSUE THREAD

**Issue by Fyzu** [nestjs/graphql#331]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

After updating 6.2.5 -> 6.4.0 on Windows, factory generate empty schema types
```
export interface IQuery {}
```
But on MacOS all its fine.

## Expected behavior

Generate correct schema types on both os (Windows, MacOS)

## Minimal reproduction of the problem with instructions

```
  const definitionsFactory = new GraphQLDefinitionsFactory()
  const clientPath = join(process.cwd(), './graphql/types/client.ts')
  await definitionsFactory.generate({
    typePaths: [
      join(process.cwd(), './graphql/core.schema.graphql'),
      join(process.cwd(), './graphql/client.schema.graphql'),
    ],
    path: clientPath,
  })
```

## Environment

<pre><code>
Nest version: 6.5.2

 
For Tooling issues:
- Node version: v12.5.0
- Platform: Windows 10
</code></pre>

**Comment by Fyzu** [comment]
@kamilmysliwiec need help :c

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue. I'll try to simulate this behavior

**Comment by Fyzu** [comment]
@kamilmysliwiec You can reproduces here: https://github.com/Fyzu/nestjs-graphql_issue331

**Comment by Fyzu** [comment]
Any updates?

**Comment by jmcdo29** [comment]
Looks like this is linked to #336 due to the fact that `fast-glob` expects Unix style paths. @kamilmysliwiec should I make a PR and add `normalize-path` to the `graphql-type.loader` file?

**Comment by kamilmysliwiec** [comment]
Fixed in 6.4.2! Thanks for reporting :)
